Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Utilities.ps1

```diff
@@ -243,6 +243,26 @@ function Get-ServiceFabricApplicationUpgradeAction
     return Get-ServiceFabricApplicationUpgrade -ApplicationName $ApplicationName
 }
 
+function Wait-ServiceFabricApplicationUpgradeAction
```

Comment:
Please think about adding L0 test for this function.

Reply:
Its a minor work but I don't want to spend any more time over this. Either I'll add it later or you take it up.

Reply:
You own this PR and hence you should decide whether to add test or not.

Reply:
Please approve if there are not any further concerns.

Reply:
If you are not adding L0, open a user story/task to track that.

Reply:
Here you go https://mseng.visualstudio.com/DefaultCollection/VSOnline/_workitems/edit/1296454